contracts/OpenSTValue.sol

```diff
@@ -72,13 +73,14 @@ contract OpenSTValue is OpsManaged, Hasher {
     uint256 private constant BLOCKS_TO_WAIT_LONG = 80667;
     // ~1hour, assuming ~15s per block
     uint256 private constant BLOCKS_TO_WAIT_SHORT = 240;
-    // indentified index position of intents mapping in storage 
+
+    // indentified index position of intents mapping in storage (in OpenSTUtility)
```

Comment:
Which intent? redemption.

Reply:
Yes redemptionIntents mapping. Updated the comment too.